### DISCUSSION TITLE
How to hide the chain icon on anchors?

### DISCUSSION THREAD

**Discussion by ooker777** [tailwindlabs/tailwindcss#13727] [Help]
In this h1 I want to hide the chain icon:
![](https://i.imgur.com/p8h46qp.png)

It has class `octicon octicon-link`. I don't know how to remove it, nor be able to find any docs on this.

**Comment by wongjn** [comment]
Could you provide some code please? Tailwind doesn't add anything like out-of-the-box, so it would be something within the project itself.

**Reply by ooker777** [reply]
Oh sorry. I thought that I was asking on the Typography repo, but somehow end up here. I didn't include the code because I thought it's the behavior of Typography. It turns out that it's from gfm. I'll ask on the correct repo though.

Here is the code btw:
```js
import { render } from "gfm";

export default function App() {
  const gfm = render("# hello");
  return (
    <div>
      <h1>Not gfm</h1>
      <div dangerouslySetInnerHTML={{ __html: gfm }} />
    </div>
  );
}
```
